api/src/http.rs

```diff
@@ -404,6 +404,7 @@ pub struct ProxyConfig {
     pub fallback: bool,
     /// Interval of P2P proxy health checking, in seconds.
     pub check_interval: u64,
```

Comment:
I'm a little concerned about the backward compatibility here.
There's no default serde function for data fields here.
Seems it may fail to parse json string missings any fields. For example, existing client may send a json config string without the "use_http" field, what happens then ?
So suggest to define default value for fields by #[serde(default = xxx_default).

Reply:
Reasonable, added serde(default) macro